recipes/dftd4/COPYING.LESSER

```diff
@@ -0,0 +1,165 @@
+                   GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
No need for these COPYING files, they'll be found from the source tarball

Reply:
Removed